index.ts

```diff
@@ -367,48 +367,312 @@ export class APIException extends Error {
     }
 }
 
+/**
+ * Error codes that App Store Server API responses return.
+ * 
+ * {@link https://developer.apple.com/documentation/appstoreserverapi/error_codes Error Codes}
```

Comment:
Please update this to use a lowercase c to match the documentation

Reply:
Thanks. I have updated this.